<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>countForm</class>
 <widget class="QWidget" name="countForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Count</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Readings:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="num_spinbox">
       <property name="toolTip">
        <string>Number of readings to take for this run.</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Delay (s):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="delay_spinbox">
       <property name="toolTip">
        <string>Time delay in seconds between successive readings</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Detectors:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="DetectorListView" name="detectors_list"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="MetadataWidget" name="metadata_widget" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="toolTip">
        <string>total exposure time for a single scan</string>
       </property>
       <property name="text">
        <string>Per scan:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DurationLabel" name="scan_duration_label">
       <property name="text">
        <string>N/A</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="toolTip">
        <string>total exposure time for a single scan</string>
       </property>
       <property name="text">
        <string>Total:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DurationLabel" name="total_duration_label">
       <property name="text">
        <string>N/A</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="toolTip">
        <string>Do this scan by multiple times</string>
       </property>
       <property name="text">
        <string>Num. of scans:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_repeat_scan_num">
       <property name="toolTip">
        <string>Do this scan by multiple times</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QueueButton" name="run_button">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QueueButton</class>
   <extends>QPushButton</extends>
   <header>firefly.queue_button</header>
  </customwidget>
  <customwidget>
   <class>DetectorListView</class>
   <extends>QListWidget</extends>
   <header>firefly.detector_list</header>
  </customwidget>
  <customwidget>
   <class>DurationLabel</class>
   <extends>QLabel</extends>
   <header>firefly.plans.duration_label</header>
  </customwidget>
  <customwidget>
   <class>MetadataWidget</class>
   <extends>QWidget</extends>
   <header>firefly.plans.metadata</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>detectors_list</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
